includes/class-ur-emailer.php

```diff
@@ -215,10 +290,10 @@ function( &$value, $key ) {
 			$message = str_replace( $to_replace, $replace_with, $message );
 			$subject = str_replace( $to_replace, $replace_with, $subject );
 
-			if ( 'yes' == get_option( 'user_registration_enable_awaiting_admin_approval_email', 'yes' ) ) {
+			if ( 'yes' === get_option( 'user_registration_enable_awaiting_admin_approval_email', 'yes' ) ) {
 				wp_mail( $email, $subject, $message, self::ur_get_header(), $attachment );
 			}
-		} elseif ( $status == - 1 ) {
+		} elseif ( - 1 === $status ) {
```

Comment:
@lekhnath005 Please verify the type of $status variable.

Reply:
@rumeshudash $status variable type is integer.